src/simulation/dynamics/Integrators/_Documentation/secModelDescription.tex

```diff
@@ -88,9 +88,14 @@ \subsubsection{4th Order Runge Kutta Fehlberg Variable Time Step}
 \end{equation}
 The updated time step is calculated through the following equation
 \begin{equation}
-	h_{\text{new}}=0.9h\left(\frac{\bm \epsilon}{\bm \delta}\right)^{1/5}, \label{eq:h_RKF45}
+	h_{\text{new}}^*=0.9h\left(\frac{\bm \epsilon_{rel}||\bm x_{n+1}|| + \bm \epsilon_{abs}}{\bm \delta}\right)^{1/5}, \label{eq:h_RKF45}
 \end{equation}
-where $\bm \epsilon$ corresponds to the relative tolerance and its default value is $10^{-4}$. If the norm of the state is smaller than the absolute tolerance (default value of $10^{-8}$), the relative error is calculated with respect to that absolute tolerance instead of the true vector norm. The time step is scaled by $0.9$ for robustness, so that the integrator will not use values that barely pass through the relative tolerance.\\
+where $\bm \epsilon_{rel}$ corresponds to the relative tolerance (with default value of $10^{-4}$), $\bm \epsilon_{abs}$ is the absolute tolerance (with default value of $10^{-8}$), and $||\bm x_{n+1}||$ is the norm of the state. The time step is scaled by $0.9$ for robustness, so that the integrator will not use values that barely pass through the tolerance.\\
```

Comment:
subscript that are name should be typeset non-italic.  Please make `\epsilon_{rel}` instead `\epsilon_{\text{rel}}`.  Do the same for `abs`.